### PR TITLE
OC-393, linter was added, linting warning where fixed

### PR DESCRIPTION
**Pull request by taaraora**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+CURRENT_DIR := $(patsubst %/,%,$(dir $(MAKEFILE_PATH)))
+
+
+define LINT
+	@echo "Running code linters..."
+	golangci-lint run -j 4 -v
+endef
+
+define GEN
+	@echo "Generationg go stubs from protobuf files"
```
Comment (sdzyba) [inline Makefile]: `Generationg` -> `Generating`

File: client.go [hunk 2]
```diff
@@ -75,6 +78,11 @@ func NewClient(cfg Config, logger logging.Logger) (*Client, error) {
 	return client, nil
 }
 
-func (c *Client) Session() *Session {
-	return c.session
+// Session returns
```
Comment (sdzyba) [inline client.go]: `Session returns ...` ? :)

File: model.go [hunk 3]
```diff
@@ -8,43 +8,62 @@ import (
...
+	// SportDescription is sent for each sport which x-feed supports.
+	// It contains list of Periods and MarketTypes which are used for specific sport
 	SportDescription struct {
 		ID          int32
 		Name        string
 		Periods     []*Period
 		MarketTypes []*MarketType
 	}
 
+	// Period represents Sport specific timespan of the game, e. g. full time, first haf, set, etc...
```
Comment (sdzyba) [inline model.go]: haf -> half

Comment (sdzyba) [reply model.go]: `etc...` -> either `etc.` or `...`

File: model.go [hunk 4]
```diff
@@ -8,43 +8,62 @@ import (
...
 	}
 
+	// PointsGroup represents statistics unit of specific type for specific game period
 	PointsGroup struct {
 		PointType     PointType
 		GroupPeriodID int32
 		State         []*State
 	}
 
+	// PointType represents type of statistics unit: Score, RedСards, Corners, etc...
```
Comment (sdzyba) [inline model.go]: etc || ...

File: session.go [hunk 5]
```diff
@@ -32,7 +33,12 @@ type Session struct {
 	entitiesMutex sync.Mutex
 }
 
+// EventsFeed returns channels of state updates for Events and Markets.
+// when there is communication errors with X-feed servers it closes the channels
```
Comment (sdzyba) [inline session.go]: there are

File: session.go [hunk 6]
```diff
@@ -32,7 +33,12 @@ type Session struct {
...
 func (s *Session) EventsFeed() (chan *EventEnvelope, chan *MarketEnvelope, error) {
+	if s == nil {
+		return nil, nil, errors.New("Session is not initialised")
```
Comment (sdzyba) [inline session.go]: session

**Review by sdzyba — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 7]
```diff
@@ -4,5 +4,9 @@ go:
   - "1.10.x"
   - tip
 
+install:
+  - go get github.com/golang/dep/cmd/dep
+  - dep ensure -vendor-only
+
```
Comment (sdzyba) [inline .travis.yml]: maybe we should add `make lint` to the CI pipeline as well?

**Review by sdzyba — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (taaraora) [reply Makefile]: thx

File: client.go (hunk 2 above)
Comment (taaraora) [reply client.go]: oops

File: .travis.yml (hunk 7 above)
Comment (taaraora) [reply .travis.yml]: makes sense

**Review by sdzyba — APPROVED** [review]

**Pull request merged**